### PR TITLE
Add Your Questions Answered page to the CSC wiki

### LINKED ISSUE #18: How does Cloud Service Certification maintain the security of the environment and the protection of the environment for open source consumers
As raised on the Cloud Service Certification call on 30th January 2020 **Ken D'Aura** from **The Hartford - Insurance Firm**

Roadmap item : How does Cloud Service Certification maintain the security of the environment and the protection of the environment for open source consumers?

### PR DESCRIPTION
**Pull request by mcleo-d**
### Description

The question **How does Cloud Service Certification maintain the security of the environment and the protection of the environment for open source consumers?** was asked by the CSC community in issue #18

This pull request add a `Add Your Questions Answered` page to the CSC wiki so relevant answers can be transferred from GitHub Issues and displayed on the wiki.

closes #18

### CODE REVIEW

File: docs/your-questions-answered.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Your Questions Answered
+
+This section of the Cloud Service Certification wiki answers questions asked by the CSC community.
+
+Ask your question by opening a [Support Issue](https://github.com/finos/cloud-service-certification/issues/new?template=Support_question.md) through GitHub.
+
+## Question
+How does Cloud Service Certification maintain the security of the environment and the protection of the environment for open source consumers? #18
```
Comment (abdullahgarcia) [inline docs/your-questions-answered.md:8]: Should we try to include the reference link to the question here? e.g. making the number the link.

File: docs/your-questions-answered.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+### Answer
+Cloud Service Certification has a peer review process to ensure the content created is functional (works as intended) and at least 2 members feel the content meets the control objective it was created for.This is OSS and it should not be trusted completely.
```
Comment (abdullahgarcia) [inline docs/your-questions-answered.md:11]: "2 members verify the content"

I think there's also a space after the "for." missing.

File: docs/your-questions-answered.md (hunk 1 above)
Comment (mcleo-d) [reply docs/your-questions-answered.md]: Great suggestion. Thanks :)

File: docs/your-questions-answered.md (hunk 2 above)
Comment (mcleo-d) [reply docs/your-questions-answered.md]: Nicely caught. Thanks!

**Comment by mcleo-d** [comment]
Hi @abdullahgarcia 

The following commits were made to resolve your feedback ...

- https://github.com/finos/cloud-service-certification/pull/52/commits/e13630331a9aa5d61f4b925b9e6d8f9066c6a991
- https://github.com/finos/cloud-service-certification/pull/52/commits/302ed5273d9561fb72a427cf43c21751659e4f11

Thanks 🚀 

James.

**Pull request merged**